M/MAGEMin/build_tarballs.jl

```diff
@@ -10,7 +10,7 @@ version = v"1.2.4"
 
 # Collection of sources required to complete build
 sources = [GitSource("https://github.com/ComputationalThermodynamics/MAGEMin", 
-                    "2bbe31fbc50827e4442ec971598ea627f3773e7b")
+                    "604ff09eb08ee2b1ac4eaa177b9c30639afb042b")
```

Comment:
Need to change also the version number, right?

Reply:
Yes, thank you, I keep on forgetting this part. My bad!

Reply:
This is a reason why I prefer the archive source: it's easier to ensure version number and actual source are in sync!